applications/potree-converter/tapisjob_app.sh

```diff
@@ -0,0 +1,13 @@
+set -x
+# Run the script with the runtime values passed in from the job request
+
+echo "Potree Converter 2.1.1"
+
+/opt/PotreeConverter/build/PotreeConverter ${converterInput} -o ${_tapisExecSystemOutputDir} -m ${samplingMethod}
+
+if [ ! $? ]; then
+      echo "PotreeConverter exited with an error status. $?" >&2
```

Comment:
We are adding the exit code to tapis file (see [here](
https://github.com/TACC/WMA-Tapis-Templates/blob/main/applications/matlab-express/tapisjob_app.sh#L3C2-L4C70) for reference):

`echo "$EXITCODE" > "${_tapisExecSystemOutputDir}/tapisjob.exitcode"`